components/AppBar/src/ColorThemer/MDCAppBarColorThemer.h

```diff
@@ -31,10 +31,25 @@
 + (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
                         toAppBar:(nonnull MDCAppBar *)appBar;
 
+/**
+ Applies a color scheme's properties to an MDCAppBar using the surface variant.
+
+ Uses the surface color as the most important color for the component.
+
+ @param colorScheme The color scheme to apply to MDCAppBar.
+ @param appBar An MDCAppBar instance to which the color scheme should be applied.
+ */
++ (void)applySurfaceVariantWithColorScheme:(nonnull id<MDCColorScheming>)colorScheme
```

Comment:
Shouldn't this be "applySurfaceVariantSemanticColorScheme"? Instead of "withColorScheme?

Maybe "applySurfaceVariantWithSemanticColorScheme"?